internal/core/window.rs

```diff
@@ -220,6 +213,53 @@ pub enum InputMethodRequest {
     },
 }
 
+/// The constraints on a window
```

Comment:
I suggest to remove the association to the window from the docs, so that we could use this elsewhere later. It'll be in `slint::LayoutConstraints`, right? Or `slint::platform::LayoutConstraints`?

```suggestion
/// This struct describes layout constraints of a resizable element, such as a window.
```